themes/custom/droopler_dorg_subtheme/droopler_dorg_subtheme.info.yml

```diff
@@ -0,0 +1,59 @@
+name: Droopler Subtheme
```

Comment:
Daj może nazwę `Droopler Subtheme for Drupal.org` i opis `Use this theme if you downloaded Droopler from drupal.org`.